projects/llm_d/orchestration/presets.d/cluster_config.yaml

```
@@ -0,0 +1,54 @@
+  cpt.kpi.labels.platform_name: CKS
+
+
+cluster_janus:
+  runtime.namespace: kpouget-dev
+
+  # Use ReadWriteMany for model cache PVC in CKS environment
+  model_cache.pvc.access_mode: ReadWriteMany
+  model_cache.pvc.storage_class_name: nfs
+  deployments.pd.resources: composite.dra.io/gpu-nic-pair
+  platform.gateway.status_address_name: null
+  platform.gateway.status_address_name: gateway-internal
```

YAMLlint (1.37.1)

[error] duplication of key "platform.gateway.status_address_name" in mapping

(key-duplicates)